### ISSUE TITLE
No longer working with Home Assistant 2023.12.0b1

### ISSUE THREAD

**Issue by PatrikTheDev** [h4de5/home-assistant-toshiba_ac#152]
I just updated to the latest Home Assistant beta (.12.0b 0 and 1) and Toshiba AC is no longer working correctly. The error:
![IMG_0229](https://github.com/h4de5/home-assistant-toshiba_ac/assets/25883885/79ce0404-ae5a-42c9-b8e9-5355db1eddaa)
![IMG_0228](https://github.com/h4de5/home-assistant-toshiba_ac/assets/25883885/4deeef21-74de-41c2-93cb-d797849743d8)

Not sure why it’s happening but I assume it’s some upstream change which is breaking toshiba_ac’s package resolution

**Comment by PatrikTheDev** [comment]
This might be related: https://github.com/home-assistant/core/issues/104851

**Comment by boozeman** [comment]
Hi,

There is the same problem with 2023.12.0 too

`2023-12-06 20:52:13.002 ERROR (MainThread) [homeassistant.setup] Setup failed for custom integration 'toshiba_ac': Requirements for toshiba_ac not found: ['toshiba-ac==0.3.3'].`

**Comment by PatrikTheDev** [comment]
I’ve opened a PR (https://github.com/KaSroka/Toshiba-AC-control/pull/42) on the Toshiba-AC-control repo which fixes this issue. I hope the maintainer finds time soon to check my changes and publish a new version.

**Comment by blbeczech82** [comment]
as quickfix, in /homeassistant/custom_components/toshiba_ac/manifest.json
change to following (not sure if version is really needed to change):
```
  "requirements": ["toshiba-ac==0.3.4"],
  "version": "2023.12.0",
```
and reboot

**Comment by Disciplus86** [comment]
> as quickfix, in /homeassistant/custom_components/toshiba_ac/manifest.json change to following (not sure if version is really needed to change):
> 
> ```
>   "requirements": ["toshiba-ac==0.3.4"],
>   "version": "2023.12.0",
> ```
> 
> and reboot

Thanks for the quick patch!

**Comment by h4de5** [comment]
should be fixed in #156
